src/MahApps.Metro/MahApps.Metro.Shared/Behaviours/BorderlessWindowBehavior.cs

```diff
@@ -1,497 +1,30 @@
-﻿using System;
-using System.Linq;
-using System.Management;
-using System.Security;
-using System.Windows;
-using System.Windows.Controls;
-using System.Windows.Interactivity;
-using System.Windows.Interop;
-using MahApps.Metro.Controls;
-using MahApps.Metro.Native;
-using Microsoft.Windows.Shell;
-
-namespace MahApps.Metro.Behaviours
+﻿namespace MahApps.Metro.Behaviours
 {
-    /// <summary>
-    /// With this class we can make custom window styles.
-    /// </summary>
-    public class BorderlessWindowBehavior : Behavior<Window>
-    {
-        private IntPtr handle;
-        private HwndSource hwndSource;
-        private WindowChrome windowChrome;
-        private PropertyChangeNotifier borderThicknessChangeNotifier;
-        private Thickness? savedBorderThickness;
-        private Thickness? savedResizeBorderThickness;
-        private PropertyChangeNotifier topMostChangeNotifier;
-        private bool savedTopMost;
-        private bool isWindwos10OrHigher;
-
-        private static bool IsWindows10OrHigher()
-        {
-            var version = Standard.NtDll.RtlGetVersion();
-            if (default(Version) == version)
-            {
-                // Snippet from Koopakiller https://dotnet-snippets.de/snippet/os-version-name-mit-wmi/4929
-                using (var mos = new ManagementObjectSearcher("SELECT Caption, Version FROM Win32_OperatingSystem"))
-                {
-                    var attribs = mos.Get().OfType<ManagementObject>();
-                    //caption = attribs.FirstOrDefault().GetPropertyValue("Caption").ToString() ?? "Unknown";
-                    version = new Version((attribs.FirstOrDefault()?.GetPropertyValue("Version") ?? "0.0.0.0").ToString());
-                }
-            }
-            return version >= new Version(10, 0);
-        }
+    using System.Windows;
+    using System.Windows.Data;
+    using ControlzEx.Behaviors;
+    using MahApps.Metro.Controls;
+    using Microsoft.Windows.Shell;
 
+    public class BorderlessWindowBehavior : WindowChromeBehavior
```

Comment:
Why should this be marked obsolete? 
Should we bind things like ResizeBorderThicknessProperty and GlowBrushProperty in InitializeStylizedBehaviors?